src/cmd/flux-mini.py

```diff
@@ -386,7 +386,9 @@ def create_parser(exclude_io=False):
         parser.add_argument(
```

Comment:
commit message for a771518b1d13476d97a1dc85860b28e5fe131621

s/unweildy/unwieldy/